app/vmalert/rule/group.go

```diff
@@ -213,7 +213,6 @@ func (g *Group) restore(ctx context.Context, qb datasource.QuerierBuilder, ts ti
 			continue
 		}
 		q := qb.BuildWithParams(datasource.QuerierParams{
-			DataSourceType:     g.Type.String(),
```

Comment:
Why this line is removed?

Reply:
We can only restore data from prometheus compatible datasource and `m.rr` is already initialized as the prometheus type.
https://github.com/VictoriaMetrics/VictoriaMetrics/blob/d5acac91c5fe5307a651a0df32e7054bfd13c0da/app/vmalert/main.go#L244-L248